cli/src/test/resources/com/devonfw/tools/ide/environment/var/devon.properties

```diff
@@ -3,5 +3,10 @@
 #************************************************************************************************
 DEVON_IDE_TOOLS=(mvn npm)
 DEVON_CREATE_START_SCRIPTS=(eclipse)
-MAVEN_VERSION=3.9.0
+MVN_VERSION=undefined
+MVN_VERSION=3.9.0
+MAVEN_VERSION=3.6.1
+DEVON_IDE_MIN_VERSION=2023.07.003
+IDE_MIN_VERSION=2024.11.001
+JAVA_VERSION=
```

Comment:
What about the case of: `MVN_VERSION=${MAVEN_VERSION}`?

Reply:
This was the other way around...
To make you happy, I changed this `devon.properties` accordingly.
However, it will not make any difference. You need to add a `EnvironmentVariablesResolvedTest` to really test this case.
Currently that does not exist and is not in scope of this story.